README.md

```diff
@@ -7,29 +7,56 @@
 
 > Share your conferences activity to your buddies
 
+## Requirements
+
+This project requires :
+
+    - Docker 
+    - mkcert (https://github.com/FiloSottile/mkcert)
+    - pipenv (https://github.com/pypa/pipenv)
+
 ## Installation
 
 ### Local
 
-> Add `local.starfleet.app` to your hosts file
+First, you will need to add local.starfleet.app to your hosts file.
+Open it with :
+```
+sudo nano /etc/hosts
+```
 
-Use [mkcert](https://github.com/FiloSottile/mkcert) to install a CA on your system to be able to generate a valid self-signed certificate for local.starfleet.app
+Then paste in `local.starfleet.app localhost 127.0.0.1`
 
-`$ mkcert *.starfleet.app`
+You now need to generate a certificate for local.starfleet.app. This is why you need mkcert, which you may find [here](https://github.com/FiloSottile/mkcert) with the installation instructions.
 
-> certificate and key must be placed in infrastructure/docker/services/router/etc/ssl/certs
+When mkcert is installed, go to your project directory and generate your certificate with :
+```bash
+mkcert -cert-file cert.pem -key-file key.pem "*.starfleet.app"
+```
 
-Install Python virtualenv to be able to use fabric commands to control easily the Docker stack
+You will find 2 certificates at the root of your project with the following names :
+> "cert.pem'
+> "key.pem'
 
-`$ pipenv install`
+Move them both to :
+`./infrastructure/docker/services/router/etc/ssl/certs/`
 
```

Comment:
Could be replaced with options provided by mkcert.

BTW, I would use inv to do that. So the readme would become

```
inv generate-certificate
```